pkg/curatedpackages/packagecontrollerclient_test.go

```diff
@@ -399,6 +405,10 @@ func TestEnableCuratedPackagesFailNoActiveBundle(t *testing.T) {
 }
 
 func TestEnableCuratedPackagesSuccessWhenApplySecretFails(t *testing.T) {
+	os.Setenv("REGISTRY_USERNAME", "admin")
+	os.Setenv("REGISTRY_PASSWORD", "Harbor12345")
+	defer os.Unsetenv("REGISTRY_USERNAME")
+	defer os.Unsetenv("REGISTRY_PASSWORD")
```

Comment:
There is a t.Setenv method as I recall that will only be used for the context of this test and no need to defer.